packages/cli/src/lib/project/project.ts

```diff
@@ -88,12 +93,24 @@ export class Project {
         archive.directory(this.resourcePath, false);
         archive.finalize();
       });
+      if (cachedManifest) {
+        writeJsonSync(this.resourceManifestPath, cachedManifest);
+      }
       return this.temporaryZipPath;
     } catch (error) {
+      if (cachedManifest) {
+        writeJsonSync(this.resourceManifestPath, cachedManifest);
+      }
       CliUx.ux.error(error as string | Error);
     }
   }
 
+  public writeResourcesManifest(orgId: string) {
+    const manifestJson =
+      readJsonSync(this.resourceManifestPath, {throws: false}) ?? {};
+    writeJsonSync(this.resourceManifestPath, {...manifestJson, orgId});
```

Comment:
Could this line throw ? 
If so, shouldn't it be handled ?

Reply:
It's quite unlikely, but yeah, let's handle it by noop its error (i.e. the manifest, it's nice, but its not the most important)